packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableEditor.ts

```diff
@@ -271,26 +280,51 @@ export default class TableEditor {
         this.editor.focus();
         if (table) {
             const vTable = new VTable(table);
-
-            const rows = vTable.cells.length - 1;
-            let lastCellIndex: number = 0;
-            vTable.cells[rows].forEach((cell, index) => {
-                if (cell.td) {
-                    lastCellIndex = index;
-                }
-            });
-
-            const selection: TableSelection = {
-                firstCell: {
-                    x: 0,
-                    y: 0,
-                },
-                lastCell: {
-                    y: rows,
-                    x: lastCellIndex,
-                },
-            };
-            this.editor.select(table, selection);
+            if (vTable.cells) {
+                const rows = vTable.cells.length - 1;
+                let lastCellIndex: number = 0;
+                vTable.cells[rows].forEach((cell, index) => {
+                    if (cell.td) {
+                        lastCellIndex = index;
+                    }
+                });
+
+                const selection: TableSelection = {
+                    firstCell: {
+                        x: 0,
+                        y: 0,
+                    },
+                    lastCell: {
+                        y: rows,
+                        x: lastCellIndex,
+                    },
+                };
+                this.editor.select(table, selection);
+            }
         }
     };
+
+    private getShouldShowTableSelectorHandler(
+        scrollContainer: HTMLElement,
+        ev?: MouseEvent
+    ): (rect: Rect) => boolean {
+        if (
+            ev?.currentTarget &&
```

Comment:
if only currentTarget is needed, just pass it only, but not the whole event ojbect